pages/_app.js

```diff
@@ -1,10 +1,35 @@
+import Navigation from "@/components/Navigation";
 import GlobalStyle from "../styles";
+import { useState } from "react";
+import plants from "@/assets/plants";
 
 export default function App({ Component, pageProps }) {
+  const [ownedPlants, setOwnedPlants] = useState([]);
+
+  function toggleOwned(plantId) {
```

Comment:
you may want to apply to a naming convention: `handleToggleOwned`. (Not a requierement but a suggestion)